test/unit/uvm-sanity.test.js

```diff
@@ -32,18 +32,18 @@ describe('uvm', function () {
             var sourceData = 'test',
                 context = uvm.spawn({
                     bootcode: `
-                        bridge.on('loopback', function (data) {
-                            bridge.dispatch('loopback', data);
+                        bridge.on('loopback2', function (data) {
```

Comment:
That is the whole issue I point out in description @czardoz. In browser mocha, somehow tests are overlapping each other and two parallel sandboxes are sending messages to all hosts due to the lack of target used and id protection. Initially @numaanashraf did mention that even if the chrome docs say otherwise, the behaviour of postMessage is confusing.

So, until we fix this issue, I renamed one of the test.
On a different note, I will rename the new test and nothing will be visible here.